Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-8: No Title of Nobility shall be granted by the United States: And no Person holding any Office of Profit or Trust under them, shall, without the Consent of the Congress, accept of any present, Emolument, Office, or Title, of any kind whatever, from any King, Prince, or foreign State.
+8: No law, increasing the compensation for the services of the Senators and Representatives, shall take effect upon a Senator or Representative who shall be, at time of passage of the law, in the respective office; nor shall any law, decreasing their compensation, take effect until a new election shall have intervened.
 
 Section 10
-1: No State shall enter into any Treaty, Alliance, or Confederation; grant Letters of Marque and Reprisal; coin Money; emit Bills of Credit; make any Thing but gold and silver Coin a Tender in Payment of Debts; pass any Bill of Attainder, ex post facto Law, or Law impairing the Obligation of Contracts, or grant any Title of Nobility.
+1: No State shall enter into any Treaty, Alliance, or Confederation; grant Letters of Marque and Reprisal; pass any Bill of Attainder, ex post facto Law, or Law impairing the Obligation of Contracts; or establish a Title of Nobility after entrance into the Union, nor shall Titles of Nobility established by a State before its entry to the Union hold or exercise authority in any other State.
 
-2: No State shall, without the Consent of the Congress, lay any Imposts or Duties on Imports or Exports, except what may be absolutely necessary for executing it's inspection Laws: and the net Produce of all Duties and Imposts, laid by any State on Imports or Exports, shall be for the Use of the Treasury of the United States; and all such Laws shall be subject to the Revision and Controul of the Congress.
+2: No State shall, without the Consent of the Congress, lay any Imposts or Duties on Imports or Exports, except what may be absolutely necessary for executing its inspection Laws: and the net Produce of all Duties and Imposts, laid by any State on Imports or Exports, shall be for the Use of the Treasury of these United States; and all such Laws shall be subject to the Revision and Control of the Congress.
 
-3: No State shall, without the Consent of Congress, lay any Duty of Tonnage, keep Troops, or Ships of War in time of Peace, enter into any Agreement or Compact with another State, or with a foreign Power, or engage in War, unless actually invaded, or in such imminent Danger as will not admit of delay.
+3: No State shall, without the Consent of Congress, lay any Duty of Tonnage, enter into any Agreement or Compact with a foreign Power, or engage in War, unless actually invaded, or in such imminent Danger as will not admit of delay.
```

Comment:
Since earlier parts of this Article have established a process by which States may enter Compacts with other States, I update that restriction only to apply to foreign powers.

The States also have the authority to maintain their militia, so they may keep troops and warships if their Citizens desire.